IanGRap/Squish
# added slow motion effect to the last player death

Contributors: Austin

Goal:
- Add a slow motion effect to the last player death

What Changed:
- MatchManager
    - When there is 1 player left, time slows, then animates back to full speed over a couple seconds

How to test:
- Load up the match scene and play some games. 
- Make sure slow motion is triggered when someone wins
- Make sure time returns back to normal eventually
- Do you like the effect?
